data-prepper-plugins/opensearch/src/main/resources/index-template/logs-otel-v1-index-standard-template.json

```diff
@@ -78,50 +149,56 @@
           "path": "observedTimestamp"
         },
         "traceId": {
-          "ignore_above": 256,
-          "type": "keyword"
+          "type": "keyword",
+          "ignore_above": 32
         },
         "spanId": {
-          "ignore_above": 256,
-          "type": "keyword"
+          "type": "keyword",
+          "ignore_above": 16
         },
         "flags": {
-          "type": "integer"
+          "type": "long"
```

Comment:
https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto#L178
Can we also add dropped_attributes_count in log

Reply:
This is defined as `fixed32`. Since this is interpreted as a bit field, I decided to use `long` to not change representation, when the highest bit is set. Do you prefer using `integer`, since only the lowest bits are currently in use?

Reply:
I am going to check, that for very attribute group there is a dropped attributes count. Thanks for pointing this out.